src/convertToObject.js

```diff
@@ -17,19 +17,19 @@ function convertToObject(sourceString) {
   const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like array , object arr, obj, result. It will decrease code readability
check it for all project

Reply:
READ CHECK LIST